includes/demo.php

```diff
@@ -35,7 +35,7 @@ function admin_bar_item( WP_Admin_Bar $wp_admin_bar ) {
 /**
  * Adds WP Notify area at the top of the dashboard
```

Comment:
Can you remove "notify" from all the comments?

Reply:
it still remains "wp-notify" in some css for custom props (and maybe few other css comments) but we will make a pr specifically for that! closing this